Specs/DateTime/English/DatePeriodParser.json

```diff
@@ -2337,4 +2337,160 @@
     }
   ]
 }
+,
+{
+  "Input": "In the 1970s",
+  "Context": {
+    "ReferenceDateTime": "2016-11-07T00:00:00"
+  },
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+    {
+      "Text": "1970s",
+      "Type": "daterange",
+      "Value": {
+        "Timex": "(1970-01-01,1979-12-31,P10Y)",
+        "FutureResolution": {
+          "startDate": "1970-01-01",
+          "endDate": "1979-12-31"
+        },
+        "PastResolution": {
+          "startDate": "1970-01-01",
+          "endDate": "1979-12-31"
+        }
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "In the 1970's",
+  "Context": {
+    "ReferenceDateTime": "2016-11-07T00:00:00"
+  },
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+    {
+      "Text": "1970's",
+      "Type": "daterange",
+      "Value": {
+        "Timex": "(1970-01-01,1979-12-31,P10Y)",
+        "FutureResolution": {
+          "startDate": "1970-01-01",
+          "endDate": "1979-12-31"
+        },
+        "PastResolution": {
+          "startDate": "1970-01-01",
+          "endDate": "1979-12-31"
+        }
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "In the 70s",
+  "Context": {
+    "ReferenceDateTime": "2016-11-07T00:00:00"
+  },
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+    {
+      "Text": "70s",
+      "Type": "daterange",
+      "Value": {
+        "Timex": "(XX70-01-01,XX79-12-31,P10Y)",
+        "FutureResolution": {
+          "startDate": "2070-01-01",
+          "endDate": "2079-12-31"
+        },
+        "PastResolution": {
+          "startDate": "1970-01-01",
+          "endDate": "1979-12-31"
+        }
+      }
+    }
+  ]
+}
+,
+{
+  "Input": "In the 70's",
```

Comment:
We should also support "'40s".

Reply:
It's already supported. There's a test case.